### DISCUSSION TITLE
Read a JSON file from the Remote Control webpage?

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/Slinger#613] [Q&A] [answered]
I would like to add some functionality to the remote control webpage.
In this sense, I've tried to read the content of JSON file which I've placed into the root of the Slingbox_Server app.

Problem: the JS code from the remote webpage cannot open it.

Question: is there a way to read from a json file?

**Comment by GerryDazoo** [comment, marked as answer]
That should be no problem. Have a look at some of the "HTML" custom remote files for examples. They may not be reading JSON formatted files but it will show you how to properly open a data file.

**Comment by bradut** [comment]
Thank you Gerry.
I have looked at these remote control examples prior to asking in this forum. Maybe something escaped me, but could't find any that reads from file.
I figured out that, in order to open a file from the remote control (HTML) form, it needs to be prefixed with "../" such as `../SlingBoxStatus.json`

It was very beneficial to me to look at these remote controls as most of them are complex and good looking.
